.pre-commit-config.yaml

```
@@ -0,0 +1,138 @@
+        entry: python -m pip audit --format json
+        language: system
+        types: [text]
+        files: ^requirements.*\.txt$
+        pass_filenames: false
+
+# Global configuration
+default_language_version:
+  python: python3.12
+
+# Hook execution order
+repos:
```

YAMLlint (1.37.1)

[error] duplication of key "repos" in mapping

(key-duplicates)